frontend/src/components/addMemberWindow.jsx

```diff
@@ -0,0 +1,75 @@
+import Button from "./button";
+import InputBox from "./inputBox";
+import ModalHeader from "./modalHeader.jsx";
+import { useHistory } from "react-router-dom";
+import { useState } from "react";
+import ModalWindow from "./modalWindow";
+import "../css/personalPage.css";
+
+function AddMemberWindow({ trigger, setTrigger }) {
+    const history = useHistory();
+    const [username, setUsername] = useState("");
+
+    const attemptAddMember = async () => {
+        if (username === "") {
+            alert("Please enter a username.");
+            return;
+        }
+
+        const response = await fetch("http://localhost:3000/v1/group/members", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+                Authorization:
+                    "bearer " + window.localStorage.getItem("authToken"),
+            },
+            body: JSON.stringify({
+                username,
+            }),
+        });
+
+        const responseJSON = await response.json();
+        if (responseJSON.username) {
+            alert("Succefully adding member");
+            history.push("/my_groups");
+        } else {
+            alert("Something went wrong with adding group member!");
+            setTrigger(false);
```

Comment:
Assuming this is a modal window (popup), on success, we should just exit the trigger window or add a small success banner (not an alert). On failure, we should alert and let them try again (so don't see the trigger to false).